mne/viz/_3d.py

```diff
@@ -415,6 +454,19 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
         View to use. See surfer.Brain().
     colorbar : bool
         If True, display colorbar on scene.
+    limits : str | 3-tuple | dict
```

Comment:
For many users (especially on the first pass through data), it seems easier just to be able to specify `limits='auto'`, and then build up from there if needed. I think being able to pass a tuple will also make it the transition easy for people sticking with the old fmin, fmid, fmax behavior.

Could the `limits` documentation use updates to make it less confusing?